pages/platform/kubernetes-k8s/vrack-example-k8s-and-pci/guide.en-gb.md

```diff
@@ -94,22 +94,39 @@ In the fourth step of creation, we attach it to the private network we created b
 
 ![Setting-up PCI - Attaching to a private network](images/vrack-example-05.png){.thumbnail}
 
-After instance creation, we can see the connection details in the OVHcloud Control Panel. If we log in to the instance using SSH, we can see that it has two network interfaces, one attached to the public IP address we use to log in, the other attached to the private network:
+After instance creation, click on the name of the new created instance, and then scroll down a little in order to see the connection details in the OVHcloud Control Panel.
 
-<pre class="console"><code>horacio@my-laptop:~$ ssh ubuntu@51.68.xxx.xxx
-Enter passphrase for key '/home/horacio/.ssh/id_rsa':
-Welcome to Ubuntu 21.04 (GNU/Linux 5.11.0-18-generic x86_64)
+![Setting-up PCI - SSH connection information](images/vrack-example-05-bis.png){.thumbnail}
 
-  System information as of Fri Jun 18 04:37:54 UTC 2021
+If we log in to the instance using SSH, we can see that it has two network interfaces, one attached to the public IP address we use to log in, the other attached to the private network:
```

Comment:
"the public IP address we use to log in" -> "the public IP address we used to log in"

Reply:
fixed